### DISCUSSION TITLE
Importação

### DISCUSSION THREAD

**Discussion by guiworldtv2** [LITUATUI/M3UPT#11] [Q&A] [answered]
Sabes como posso importar esta lista de canais para a minha lista, sem configurar no aplicativo uma nova lista, apenas jogando este link dentro do meu link m3u?
não entendo muito sobre iptv e estou a usar kodi no firetv stick

**Comment by LITUATUI** [comment, marked as answer]
Olá.

Podes usar o IPTV Merge.

https://forum.kodi.tv/showthread.php?tid=340691
